config/src/test/java/org/springframework/security/config/annotation/method/configuration/PrePostMethodSecurityConfigurationTests.java

```diff
@@ -998,6 +998,14 @@ public void adviseWhenPrePostEnabledThenEachInterceptorRunsExactlyOnce() {
 		verify(expressionHandler, times(4)).createEvaluationContext(any(Supplier.class), any());
 	}
 
+	@Test
```

Comment:
Since this is for a bug fix, will you please place a comment indicating the bug that this fixes? Like so:

```
// gh-12345
@Test
...
```